devicetypes/smartthings/ajax-rgbcw-strip.src/ajax-rgbcw-strip.groovy

```diff
@@ -0,0 +1,276 @@
+/**
+ *  Copyright 2017 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Author: SmartThings
+ *  Date: 2016-01-19
+ *
+ *  This DTH should serve as the generic DTH to handle RGBW ZigBee HA devices
+ */
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition (name: "Ajax RGBCW STRIP", namespace: "smartthings", author: "SmartThings", runLocally: true, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true, genericHandler: "Zigbee") {
```

Comment:
Don't create new DTH, add fingerprint of your device to ZigBee RGBW Bulb (on which this DTH is based, I suppose)